projects/01-pair-matching/hadi-mobarra/webpack/src/index.js

```diff
@@ -0,0 +1,106 @@
+import './styles.css';
+
+// Fetch API to get the number of row and column
+async function chartNumberFunc() {
+  const chartNumber = await fetch('http://192.168.43.126:8000/playgame')
+    .then((response) => response.json())
+    .then((data) => data.dimension)
+    .catch((err) => (err));
+  return chartNumber;
+}
+
+// Array of numbers for Game
+const gameValues = [];
+for (let i = 1; i <= 32; i += 1) {
+  gameValues.push(i);
+  gameValues.push(i);
+}
+
+// shuffle Array
+gameValues.sort(() => 0.5 - Math.random());
+const card = document.createElement('div');
+card.setAttribute('id', 'game');
+document.getElementById('box').appendChild(card);
+const newHalfTwo = gameValues.slice();
+
+let choosenCards = [];
+let choosenId = [];
+let trueChoosenCards = [];
+// const cardsInGame = 32;
+
+// Checks if two values of Table are equal or not
+function cardMatcher() {
+  if (choosenCards.length !== 2) {
+    const cardID = this.getAttribute('card-id');
+    const inner = this.innerHTML;
+    const innerHtml = this.getAttribute('inner');
+    if (inner !== 'True') {
+      choosenCards.push(innerHtml);
+      choosenId.push(cardID);
+      if (choosenCards.length === 2) {
+        setTimeout(() => {
+          const firstCard = choosenCards[0];
+          const secondCard = choosenCards[1];
+          if (firstCard === secondCard) {
+            trueChoosenCards += trueChoosenCards;
+            // Changes the value of found numbers to its own value.
+            const divs2 = document.getElementsByTagName('div'); let
+              i = divs2.length;
+            while (i >= 0) {
+              i -= 1;
+              if (divs2[i].getAttribute('inner') === choosenCards[0]) {
+                divs2[i].innerHTML = firstCard;
+              }
+            }
+            document.getElementById('Trues').innerHTML = trueChoosenCards;
+            // alert('Trueeee :)');
+          }
+          // make arrays empty for the next step
+          choosenCards = [];
+          choosenId = [];
+          // wining message
+          // if (cardsInGame == trueChoosenCards) {
+          //  alert('You Won!');
+          // }
+        }, 300);
+      }
+    }
+  }
+}
+
+// Generate Table with 8*8 squares
```

Comment:
Fix this comment about the number of squares and remove other commented codes please.